docker/build_and_test.sh

```diff
@@ -6,4 +6,8 @@ npm run compile
 # disabled while promise rejections not returning error objects need to be addressed
 # npm run lint
 npm test -- --forbid-only --forbid-pending
+# prevent publish of any pacts with 'to-be' in their name
+for i in ./pacts/*-to-be-*.json; do mv "$i" "${i%.json}.ignore"; done
 npm run publish-pacts
+# restore 'to-be' pacts
+for i in ./pacts/*.ignore; do mv "$i" "${i%.ignore}.json"; done
```

Comment:
do we need to restore the pacts?
can you verify the publish pact job re-creates a new /pact folder on each build - if so we dont need to restore the 'to-be' pacts.

Reply:
My original rationale was that the pacts may be needed for stub purposes later on, but these happen outside of the docker container, so you're right. I'll remove the restore as it achieves nothing.